metadata/src/main/java/org/apache/kafka/controller/QuorumController.java

```diff
@@ -1914,6 +1915,7 @@ public CompletableFuture<List<CreatePartitionsTopicResult>> createPartitions(
         });
     }
 
+    // TODO: Figure out the reason as to why is a snapshot starting here? Who is calling beginWritingSnapshot()?
```

Comment:
This method is used in tests, please also remove this comment.

Reply:
Hi @dengziming, forgot to remove this comment. I have removed it now.
Thanks